src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java

```diff
@@ -71,11 +76,24 @@ public class InterestServiceImpl implements InterestService {
   @Value("${search.result.redis.ttl}")
   private long searchResultRedisTtl;
 
+  @Autowired
+  private CassandraOperation cassandraOperation;
+
   @Override
   public CustomResponse createInterest(JsonNode interestDetails) {
     log.info("InterestServiceImpl::createInterest:entered the method: "+interestDetails);
     CustomResponse response = new CustomResponse();
     payloadValidation.validatePayload(Constants.INTEREST_VALIDATION_FILE_JSON,interestDetails);
+    Map<String, Object> propertyMap = new HashMap<>();
+    propertyMap.put(Constants.ID,interestDetails.get(Constants.ORG_ID).asText());
+    List<Map<String, Object>> orgDetails = cassandraOperation.getRecordsByPropertiesWithoutFiltering(
+        Constants.KEYSPACE_SUNBIRD, Constants.ORG_TABLE, propertyMap, null, 1);
+    if(ObjectUtils.isEmpty(orgDetails)){
```

Comment:
@ruksana2808 please do the proper formating and use collectionUtils as it a list